metric/memory.go

```diff
@@ -4,29 +4,25 @@ import (
 	"log"
 
 	uuid "github.com/satori/go.uuid"
-	"github.com/shirou/gopsutil/mem"
 	"github.com/thundra-io/thundra-lambda-agent-go/application"
 )
 
+const miBToB = 1024 * 1024
+
 func prepareMemoryMetricsData(mp *metricPlugin, base metricDataModel) metricDataModel {
 	base.ID = uuid.NewV4().String()
 	base.MetricName = memoryMetric
 
-	memInfo, err := mem.VirtualMemory()
-	if err != nil {
-		log.Println(err)
-	}
-
 	procMemInfo, err := proc.MemoryInfo()
 	if err != nil {
 		log.Println(err)
 	}
+	
+	application.MemoryUsed = int(procMemInfo.RSS / miBToB)
```

Comment:
Are we sure that RSS is in bytes? Not sure but as far as I remember, it is reported in KB

Reply:
I've tried it on lambda and also debugged it, and the results were as expected.

Reply:
Ok :+1: